src/mame/taito/gladiatr.cpp

```diff
@@ -231,21 +231,21 @@ void gladiatr_state_base::ym_irq(int state)
 void gladiatr_state::gladiator_adpcm_w(u8 data)
 {
 	// bit 6 = bank offset
-	membank("bank2")->set_entry((data & 0x40) ? 1 : 0);
+	m_adpcmbank->set_entry(BIT(data, 6) ? 1 : 0);
```

Comment:
This is somewhat redundant now – if you’re using the `BIT` helper, you can just do `m_adpcmbank->set_entry(BIT(data, 6));`